### ISSUE TITLE
Edit title element for your app

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#22]
## Description
Today the title of an app is made up of the app-owner name (Not full name!) and the repository name from Gitea. The idea behind these values are good, but they are probably not always good enough. And with naming restrictions on repository name it could be hard to read for a screen reader. This element is visible in the browser tabs and are read by a screen reader. 

Example: ttd- ui-components

This is a WCAG issue
[Tilsynet(Norwegian)](https://www.uutilsynet.no/regelverk/testprosedyrar-nettstader/709#suksesskriterium_242_sidetitlar)
[WCAG guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)

**Proposed standard title**
Application title and org-full name

## Considerations
* can we also support different languages? 

## Acceptance criteria
> Describe criteria here (i.e. What is allowed/not allowed (negative tesing), validations, error messages and warnings etc.)

## Specification tasks

- [ ] Development tasks are defined

## Development tasks
> Add tasks here

**Comment by Febakke** [comment]
@bjosttveit I think its enough with the proposed standard title as a start.

**Comment by bjosttveit** [comment]
https://altinn.slack.com/archives/C02ESDSGPN1/p1673959846896719
